src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -70,9 +71,11 @@ public PersistentDataStore() {
         String userName = (String) entity.getProperty("username");
         String passwordHash = (String) entity.getProperty("password_hash");
         String bio = (String) entity.getProperty("biography");
+        ArrayList<String> interests = (ArrayList<String>) entity.getProperty("interests");
```

Comment:
does this work? if you restart your server are interests preserved? thinking you'll have to also use Arrays.asList to go from a string (returned by getProperty) and convert it to an arraylist

Reply:
I tested it by relaunching the server a few times and it seemed to all load correctly. I was a bit unsure because of the warnings though. I will try converting it from a string again though and see if it works better